##########################
# wso2am-gateway_manager #
##########################

repository/conf/api-manager.xml
===============================

Ignore this diff will Gives
    AMQConnection Unable to connect to broker api manager

@@ -460,7 +460,7 @@
         </WhiteListedURIs>
     </RESTAPI>
     <ThrottlingConfigurations>
-        <EnableAdvanceThrottling>true</EnableAdvanceThrottling>
+        <EnableAdvanceThrottling>false</EnableAdvanceThrottling>
         <DataPublisher>
             <Enabled>true</Enabled>
             <Type>Binary</Type>

Rejected diff
@@ -508,14 +508,14 @@
                 <JobQueueSize>10</JobQueueSize>
             </JMSTaskManager>
         </JMSConnectionDetails>
-        <!--<JMSEventPublisherParameters>-->
-                <!--<java.naming.factory.initial>org.wso2.andes.jndi.PropertiesFileInitialContextFactory</java.naming.factory.initial>-->
-                <!--<java.naming.provider.url>repository/conf/jndi.properties</java.naming.provider.url>-->
-                <!--<transport.jms.DestinationType>topic</transport.jms.DestinationType>-->
-                <!--<transport.jms.Destination>throttleData</transport.jms.Destination>-->
-                <!--<transport.jms.ConcurrentPublishers>allow</transport.jms.ConcurrentPublishers>-->
-                <!--<transport.jms.ConnectionFactoryJNDIName>TopicConnectionFactory</transport.jms.ConnectionFactoryJNDIName>-->
-        <!--</JMSEventPublisherParameters>-->
+        <JMSEventPublisherParameters>
+                <java.naming.factory.initial>org.wso2.andes.jndi.PropertiesFileInitialContextFactory</java.naming.factory.initial>
+                <java.naming.provider.url>repository/conf/jndi.properties</java.naming.provider.url>
+                <transport.jms.DestinationType>topic</transport.jms.DestinationType>
+                <transport.jms.Destination>throttleData</transport.jms.Destination>
+                <transport.jms.ConcurrentPublishers>allow</transport.jms.ConcurrentPublishers>
+                <transport.jms.ConnectionFactoryJNDIName>TopicConnectionFactory</transport.jms.ConnectionFactoryJNDIName>
+        </JMSEventPublisherParameters>
         <!--DefaultLimits>
             <SubscriptionTierLimits>
                 <Gold>5000</Gold>